run_uwsgi.sh

```diff
@@ -22,4 +22,9 @@ fi
 # Make sure directories that are shared are created
 mkdir -p /var/www/images/_upload/{0..9} && chmod 777 -R /var/www/images/_upload
 
-uwsgi uwsgi.ini
+# grep -Fxq seems not working...
+[ $(awk 'BEGIN{ok=0}/PLUGINS_ENABLED/,/]/{if (!/#/&&/remote_build/) ok=1}END{print ok}' \
+/code/shub/settings/config.py) -eq 0 ] && uwsgi uwsgi.ini ||
+# Add support to websocket server, Daphne, throught django channels 
```

Comment:
This cannot be done through Python somewhere in the application?

Reply:
I don't find till now another way to proceed.
I'm writing specific documentation regarding this important part...